datafusion/sqllogictest/test_files/functions.slt

```diff
@@ -479,7 +479,7 @@ CREATE TABLE test(
 (3, 30);
 
 # Scalar function
-statement error Did you mean 'arrow_typeof'?
+statement error Invalid function 'arrowtypeof'
```

Comment:
I think this is tracked by https://github.com/apache/arrow-datafusion/issues/9392

@SteveLauC  has a PR up for it https://github.com/apache/arrow-datafusion/pull/9407 which hopefully will land shortly

Reply:
https://github.com/apache/arrow-datafusion/pull/9407 is ready to go -- I think once we merge that one we can revert this change

Reply:
3dbafa70e